data/youtube.js

```diff
@@ -1,14 +1,11 @@
 import axios from 'axios';
 import Keys from '../constants/Keys';
 
-const TARGET_SECTION_TITLE = "Popular Series"
-
-const PLAYLIST_PARTS = ['id', 'contentDetails', 'snippet'];
```

Comment:
each call doesn't require all of these parts, so we can just explicitly define them in the params